src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5497,4 +5497,10 @@
   <data name="WRN_Experimental_Title" xml:space="preserve">
     <value>Type is for evaluation purposes only and is subject to change or removal in future updates.</value>
   </data>
-</root>
+  <data name="FEATURE_InterpolatedStrings" xml:space="preserve">
+    <value>interpolated strings</value>
+  </data>
+  <data name="ERRWRN_Last_Title" xml:space="preserve">
```

Comment:
I remove it and retest, as the there is another unit test that fails without it.